### PR TITLE
ETV-1575-use-sdk

### PR DESCRIPTION
**Pull request by alsofronie**
- [x] Added `@etvas/etvas-sdk` to server stack.
- [x] Added examples for etvas events in server stack
- [x] Refactor calls to Etvas Platform backend to use the configured client from `etvas/etvas-sdk`
- [x] Fixed high vulnerability in client (`npm audit fix`)
- [x] Up-to-date documentation

### CODE REVIEW

File: server/etvas-events/product-canceled.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+/**
+ * Product Canceled handler function
+ *
+ * @param {object} payload The event payload
+ * @returns True or object
+ */
+const handler = async ({ userId, purchaseId }) => {
+  // If you previously stored your internal ID in ETVAS
+  // platform, you can retrieve and co a cleanup on your side
+  const { internalId } = etvas.client.read(purchaseId)
```
Comment (ohnu) [inline server/etvas-events/product-canceled.js:26]: let's mention you can also clean up based on purchaseId

File: server/etvas-events/product-canceled.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/**
+ * This handler is called when the product.canceled event is emitted
+ * in ETVAS Platform. The app receives this event as a POST request,
+ * on the URL provided in Platform at product setup. The SDK
+ * automatically routes the event to this handler, which receives
+ * the payload as an object containing userId and purchaseId. Both
```
Comment (ohnu) [inline server/etvas-events/product-canceled.js:6]: we are not sending the user id

```
  readonly productId: string
  readonly purchaseId: string
```

File: server/etvas-events/product-canceled.js (hunk 1 above)
Comment (alsofronie) [reply server/etvas-events/product-canceled.js:26]: It's on line 34

Comment (ohnu) [reply server/etvas-events/product-canceled.js:26]: that is something else .. that clears data on etvas .. 
usually we expect integrations to cleanup with 

`await myApi.removeSubscription(purchaseId)`

**Pull request merged**